app/(main)/manager/discography/page.jsx

```
@@ -5,49 +5,27 @@
+        <div className="flex flex-col gap-4">
+          <div className="flex flex-col gap-4">
+            <h2 className="text-xl font-bold">Albums</h2>
+          </div>
+          <div className="flex flex-col gap-4">
+          </div>
         </div>
       </div>
+      <button
+        className="absolute bottom-4 right-4 p-0 w-16 h-16 bg-blue-500 rounded-full text-white text-3xl flex items-center justify-center"
+        onClick={handleFabClick}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)